wd_helper.au3

```diff
@@ -961,7 +985,13 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 		Else
 			$sReadyState = _WD_ExecuteScript($sSession, 'return document.readyState', '', Default, $_WD_JSON_Value)
 			$iErr = @error
-			If $iErr Or $sReadyState = 'complete' Then
+			If $iErr Then
+				$iErr = $_WD_ERROR_Exception
+				$sReadyState = ''
+				ExitLoop
+			EndIf
+
+			If $sReadyState = 'complete' Then
```

Comment:
Elseif ?

Reply:
I considered it, but decided not to make the change. I believe the code with work functionally the same either way.

Reply:
J